src/main/java/net/snowflake/client/core/HttpClientSettingsKey.java

```diff
@@ -46,6 +49,27 @@ public HttpClientSettingsKey(OCSPMode mode) {
     this.ocspMode = mode != null ? mode : OCSPMode.FAIL_OPEN;
   }
 
+  public HttpClientSettingsKey(
```

Comment:
If we're pushing for builder pattern then we should make new constructors private or at least protected

Reply:
Made it package private